skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -365,6 +365,40 @@ void MpActor::MpApiDeath(MpActor* killer)
   }
 }
 
+void MpActor::ModifyActorValue(espm::ActorValue av, float value)
+{
+  if ((av == espm::ActorValue::Health || av == espm::ActorValue::Stamina ||
+       av == espm::ActorValue::Magicka) == false) {
+    return;
+  }
+  BaseActorValues base =
+    GetBaseActorValues(GetParent(), GetBaseId(), GetRaceId());
```

Comment:
please create and use `GetMaximumValues` as alias of `GetBaseValues`.
in the future magic, effects will affect `GetMaximumValues`. currently, both are the same.